pages/linux/speedometer.md

```diff
@@ -0,0 +1,8 @@
+# speedometer
+
+> Python script that shows network traffic graph in terminal.
```

Comment:
Perhaps this will read better?
```suggestion
> Python script that shows a network traffic graph in the terminal.
```

Reply:
resolved in [https://github.com/tldr-pages/tldr/pull/4267/commits/2e7ada923eaa1692248c337b498018bc58f33ce2 ](https://github.com/tldr-pages/tldr/pull/4267/commits/2e7ada923eaa1692248c337b498018bc58f33ce2)